jbpm-test-coverage/src/test/resources/org/jbpm/test/functional/event/BoundaryConditionalProcessWithOutputSet.bpmn2

```diff
@@ -0,0 +1,116 @@
+
+	<property id="nodeInstanceProcess" itemSubjectRef="item_nodeInstance_ref_process"/>
+	<property id="signalProcess" itemSubjectRef="item_signal_ref_process"/>
+	<property id="eventProcess" itemSubjectRef="item_event_ref_process"/>
+	
+    <!-- nodes -->
+    <startEvent id="_1" name="StartProcess" />
+
+      <userTask id="_2" name="Human Task" tns:taskName="Human Task" >
+        <ioSpecification>
+          <inputSet></inputSet>
+          <outputSet></outputSet>
+        </ioSpecification>
+      </userTask>
+      
+  
+    <endEvent id="_3" name="End Main Process">
+      <terminateEventDefinition/>
+    </endEvent>
+  
+    <boundaryEvent id="_6" name="Timer Event" attachedToRef="_2" cancelActivity="false">
+      <conditionalEventDefinition id="_AzlHbBoLEeu2L4IddwlS5w">
+        <condition xsi:type="tFormalExpression" id="FormalExpression_1" language="http://www.jboss.org/drools/rule">org.jbpm.test.entity.Person(fullName == &quot;john&quot;)</condition>
+      </conditionalEventDefinition>
+  	  <dataOutput id="nodeInstanceRef" name="nodeInstance" />
+  	  <dataOutput id="signalRef" name="signal" />
+  	  <dataOutput id="eventRef" name="event" />
+      <outputSet>
+        <dataOutputRefs>nodeInstanceRef</dataOutputRefs>
+        <dataOutputRefs>signalRef</dataOutputRefs>
+        <dataOutputRefs>eventRef</dataOutputRefs>
+      </outputSet>
+      <dataOutputAssociation>
+        <sourceRef>nodeInstanceRef</sourceRef>
+        <targetRef>nodeInstanceProcess</targetRef>
+      </dataOutputAssociation>
+      <dataOutputAssociation>
+        <sourceRef>signalRef</sourceRef>
+        <targetRef>signalProcess</targetRef>
+      </dataOutputAssociation>
+      <dataOutputAssociation>
+        <sourceRef>eventRef</sourceRef>
+        <targetRef>eventProcess</targetRef>
+      </dataOutputAssociation>
+    </boundaryEvent>
+  
+  	<task id="_7" drools:taskName="DataCaptureTask">
+  		<ioSpecification>
+			<dataInput id="nodeInstanceRef" name="nodeInstance"/>
+			<dataInput id="signalRef" name="signal"/>
+			<dataInput id="eventRef" name="event"/>
+			<dataInput id="workItemRef" name="workItem"/>
+			<inputSet>
+		  		<dataInputRefs>nodeInstanceRef</dataInputRefs>
+		  		<dataInputRefs>signalRef</dataInputRefs>
+		  		<dataInputRefs>eventRef</dataInputRefs>
+		  		<dataInputRefs>workItemRef</dataInputRefs>
+			</inputSet>
+			<outputSet>
+			</outputSet>
+	    </ioSpecification>
+		<dataInputAssociation>
+			<sourceRef>nodeInstanceProcess</sourceRef>
+			<targetRef>nodeInstanceRef</targetRef>
+		</dataInputAssociation>
+		<dataInputAssociation>
+			<sourceRef>signalProcess</sourceRef>
+			<targetRef>signalRef</targetRef>
+		</dataInputAssociation>
+	    <dataInputAssociation>
+			<sourceRef>eventProcess</sourceRef>
+			<targetRef>eventRef</targetRef>
+		</dataInputAssociation>
+		<dataInputAssociation>
+			<sourceRef>workItemProcess</sourceRef>
+			<targetRef>workItemRef</targetRef>
+		</dataInputAssociation>
```

Comment:
`workItemProcess` is not defined in this process

Reply:
added.